webapp/src/components/kanban/kanban.tsx

```diff
@@ -55,7 +56,19 @@ const vStrength = createVerticalStrength(Utils.isMobile() ? 60 : 250)
 
 const Kanban = (props: Props) => {
     const currentView = useAppSelector(getCurrentView)
+    const cardTemplates: Card[] = useAppSelector(getCurrentBoardTemplates)
     const {board, activeView, cards, groupByProperty, visibleGroups, hiddenGroups, hiddenCardsCount} = props
+    const [defaultTemplateID, setDefaultTemplateID] = useState('')
```

Comment:
If state isn't set, I would prefer it be empty, rather than empty string. Will require change to line 313.
```suggestion
    const [defaultTemplateID, setDefaultTemplateID] = useState<string>();
```

Reply:
Okay